Improve documentation for SearchAfter field (#2687)
camunda/camunda-docs: docs/apis-tools/tasklist-api-rest/schemas/requests/task-search-request.mdx

#### [<code style={{ fontWeight: 'normal' }}>TaskSearchRequest.<b>searchAfter</b></code>](#)<Bullet />`string` <Badge class="secondary" text="list"/>

> > Array of values that should be copied from `sortValues` of one of the tasks from the current search results page.

Reviewer: ```suggestion
> Used to return a paginated result. Array of values that should be copied from `sortValues` of one of the tasks from the current search results page.
```

Let's add this sentence to all the 4 options to make the purpose of it clear, and allow users to find quickly how to paginate with the API (it's a common question)

Author: updated for the 4 fields

> > It enables the API to return a page of tasks that directly follow the provided values in terms of sorting order.

Reviewer: ```suggestion
> It enables the API to return a page of tasks that directly follow the task identified by the provided values, with respect to the sorting order.
```

Let's please add this to all the 4 options, I believe it makes it clearer

Author: updated for the 4 fields
